packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -685,6 +689,8 @@ describe('<Tabs />', () => {
 
         await waitFor(() => {
           expect(hasLeftScrollButton(container)).to.equal(true);
+        });
+        await waitFor(() => {
```

Comment:
Only one `expect` per `waitFor`. Related rule https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-wait-for-multiple-assertions.md